src/main/java/ch/njol/skript/expressions/LitPi.java

```diff
@@ -0,0 +1,59 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleLiteral;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Pi")
+@Description("Returns the mathematical constant pi. (approx. 3.1415926535")
+@Examples("set {_tau} to pi * 2")
+@Since("INSERT VERSION")
+public class LitPi extends SimpleLiteral<Number> {
```

Comment:
```suggestion
public class LitPi extends SimpleLiteral<Double> {
```
The return type could be more specific, Use `Double` instead

Reply:
Skript only uses Long for ints and Number for decimal, and I am not deviating from that style.

Reply:
No Skript supports all Number types. You're supposed to return the correct type on output. Syntaxes typically only take %number% as the input because that's how it's been as it's easier for the Skript user to understand rather than every number type being the correct type.

Reply:
Okay good to know, I'll change it then.